python/lsst/analysis/drp/dataSelectors.py

```diff
@@ -1,5 +1,6 @@
-__all__ = ("FlagSelector", "PsfFlagSelector", "StarIdentifier", "GalaxyIdentifier", "UnknownIdentifier",
-           "CoaddPlotFlagSelector", "BaseSNRSelector", "SnSelector")
+__all__ = ("FlagSelector", "PsfFlagSelector", "BaseSNRSelector", "SnSelector",
```

Comment:
I'm not quite sure about exposing BaseSNRSelector here. It doesn't seem to be used anywhere(?) and if it is used, it would be to define other SNR selectors.

Reply:
I noticed that it is unused too and plan to address that on another ticket (I first want to check with the original author that removing it is the right thing to do).  Since it was already in the list, I'll leave it as is here and deal with it on the removal ticket.